Source/OCMock/OCMock.h

```diff
@@ -22,6 +22,7 @@
 #import <OCMock/OCMLocation.h>
 #import <OCMock/OCMMacroState.h>
 #import <OCMock/NSNotificationCenter+OCMAdditions.h>
+#import <OCMock/NSInvocation+OCMAdditions.h>
```

Comment:
Double check that this file doesn't cause anything crazy and that's why it was excluded by the library authors

Reply:
From this [pr](https://github.com/erikdoe/ocmock/commit/73bd007d7f6cfd78236b447382e2a4705065848a#diff-b7f72cf8f6a6a3ae89cd54e5af1210b8), it looks like they forgot to make it public. But the goal is to preserve which files we are using in EMA.